libvirt/tests/src/numa/numa_invalid_nodeset.py

```diff
@@ -0,0 +1,57 @@
+import logging
+
+from virttest import libvirt_xml
+from virttest import utils_misc
+from virttest import utils_test
+from virttest import virsh
+
+
+def update_xml(vm_name, params):
+    numa_info = utils_misc.NumaInfo()
+    online_nodes = numa_info.get_all_nodes()
+    # Prepare a memnode list
+    numa_memnode = [{
+        'mode': params.get('memory_mode', 'strict'),
+        'cellid': params.get('cellid', '0'),
+        # Take a first node above available nodes.
+        'nodeset': params.get('memory_nodeset', str(int(online_nodes[-1]) + 1))
+    }]
+    # Prepare a numa cells list
+    numa_cells = [{
+        'id': '0',
+        # cpus attribute is optional and if omitted a CPU-less NUMA node is
+        # created as per libvirt.org documentation.
+        'memory': '512000',
+        'unit': 'KiB'}
+    ]
+    vmxml = libvirt_xml.VMXML.new_from_dumpxml(vm_name)
+    del vmxml.numa_memory
+    vmxml.numa_memnode = numa_memnode
+    vmcpuxml = libvirt_xml.vm_xml.VMCPUXML()
+    vmcpuxml.xml = "<cpu mode='host-model'><numa/></cpu>"
+    logging.debug(vmcpuxml.numa_cell)
+    vmcpuxml.numa_cell = vmcpuxml.dicts_to_cells(numa_cells)
+    logging.debug(vmcpuxml.numa_cell)
+    vmxml.cpu = vmcpuxml
```

Comment:
Would you like to try this way? 
I have compare the output of vmxml, they're the same.

```suggestion
    vmxml.setup_attrs(**{
        'numa_memnode': numa_memnode,
        'cpu': {'reset_all': True, 'mode': 'host-model', 'numa_cell': numa_cells}
    })
```